app/src/test/java/mozilla/lockbox/store/TelemetryStoreTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.store
+
+import android.content.Context
+import io.reactivex.subjects.ReplaySubject
+import mozilla.lockbox.DisposingTest
+import mozilla.lockbox.action.TelemetryAction
+import mozilla.lockbox.action.TelemetryEventMethod
+import mozilla.lockbox.action.TelemetryEventObject
+import mozilla.lockbox.flux.Dispatcher
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.telemetry.event.TelemetryEvent
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(packageName = "mozilla.lockbox")
+class TelemetryStoreTest : DisposingTest() {
+    class FakeTelemetryWrapper : TelemetryWrapper() {
+        val applySubject: ReplaySubject<Context> = ReplaySubject.create(1)
+        val eventsSubject: ReplaySubject<TelemetryEvent> = ReplaySubject.create(1)
+
+        override val ready: Boolean get() = true
+        override fun apply(ctx: Context) {
+            applySubject.onNext(ctx)
+        }
+        override fun recordEvent(event: TelemetryEvent) {
+            eventsSubject.onNext(event)
+        }
+    }
+
+    private lateinit var dispatcher: Dispatcher
+    private lateinit var wrapper: FakeTelemetryWrapper
+    private lateinit var subject: TelemetryStore
+
+    @Before
+    fun setUp() {
+        dispatcher = Dispatcher()
```

Comment:
This might be more of a nit / implementation detail / does it actually matter I dunno... but in the other `Test`s we just do this initialization inline with the variable declarations. However I can see the utility of initialization in `@Before` if you are explicitly testing initialization behavior.

Reply:
I was copying from DataStore, which does have some initialization tests.  I can move this out for this test.